github/ci/prow/files/config.yaml

```diff
@@ -275,7 +275,7 @@ branch-protection:
               protect: true
               required_status_checks:
                 contexts:
-                  - continuous-integration/travis-ci/pr
+                  # - continuous-integration/travis-ci/pr
```

Comment:
Maybe rather just delete the line, git will remember it just in case :)

Reply:
Done